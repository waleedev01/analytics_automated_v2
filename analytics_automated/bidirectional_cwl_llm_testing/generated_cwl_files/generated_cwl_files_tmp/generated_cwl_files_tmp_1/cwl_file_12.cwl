cwlVersion: v1.0

cwl:echo
label: Echo
inputs:
  - id: message
    type: string
outputs:
  - id: output
    type: string