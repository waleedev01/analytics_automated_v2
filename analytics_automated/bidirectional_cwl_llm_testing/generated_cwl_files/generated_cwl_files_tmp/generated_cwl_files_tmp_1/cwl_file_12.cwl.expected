{"is_valid": false, "error": "Expected error message"}